.github/workflows/manual-publish.yml

```diff
@@ -31,8 +33,23 @@ jobs:
           ssm_parameter_pairs: '/production/common/releasing/pypi/token = PYPI_AUTH_TOKEN'
 
       - uses: ./.github/actions/build
+        id: build
 
       - uses: ./.github/actions/publish
         with:
           token: ${{env.PYPI_AUTH_TOKEN}}
           dry_run: ${{ inputs.dry_run }}
+
+  release-provenance:
+    needs: [ 'build-publish' ]
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      id-token: write
+      contents: write
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v1.7.0
+    with:
+      base64-subjects: "${{ needs.build-publish.outputs.package-hashes }}"
+      upload-assets: true
+      upload-tag-name: TBD
```

Comment:
TBD should be fixed here.

Reply:
Since we're triggering manual publishes in the case that a `release-please` workflow fails, the provenance generator should by default use the tag associated with the branch HEAD that we're building off of, so I think we can safely leave that parameter off here.